WordPress/Classes/Utility/EmailCheckerHelper.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+@interface EmailCheckerHelper : NSObject
+
+/**
+ A proxy for the EmailChecker pod. Allows the EmailChecker functionality to be
+ used in .swift files. 
+ */
++ (NSString *) suggestDomainCorrection:(NSString *)email;
```

Comment:
I have a Swift version of EmailChecker at https://github.com/koke/EmailChecker-ios
Should we just make the switch? 😁